aten/src/ATen/native/cudnn/ConvShared.cpp

```diff
@@ -491,6 +491,77 @@ Tensor cudnn_convolution_transpose_backward_weight(
       padding, stride, dilation, groups, benchmark, deterministic, allow_tf32);
 }
 
+Tensor cudnn_convolution_add_relu(
+    const Tensor& input_t,
+    const Tensor& weight_t,
+    const optional<Tensor>& bias_t,
+    const optional<Tensor>& z_t,
+    optional<Scalar> alpha_t,
+    IntArrayRef padding,
+    IntArrayRef stride,
+    IntArrayRef dilation,
+    int64_t groups) {
+  // Skip all the shape and type checking because frozen-model opt should
+  // catch problems during the compilation
```

Comment:
We should actually add this logic to the fusion pass bc it doesnt exist right now